packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/Columns.js

```diff
@@ -16,6 +16,10 @@ import update from 'immutability-helper';
 import { pkg } from '../../../../../settings';
 import DraggableElement from '../../DraggableElement';
 import { isColumnVisible } from './common';
+// import cx from 'classnames';
+// import { Draggable16 } from '@carbon/icons-react';
+// import { RowSizeDropdown } from '../RowSize';
+// import { elementOrParentIsFloatingMenu } from '../../../../../global/js/utils/wrapFocus';
```

Comment:
commented out code should really be cleaned out (there are of course exceptions).